persist-cli/src/main/java/io/ballerina/persist/utils/BalProjectUtils.java

```diff
@@ -85,4 +95,69 @@ private static String fotmatError(String errorMessage, Path path) {
         }
         return errorMessage;
     }
+
+    public static EntityMetaData readBalFiles(String sourcePath) throws BalException {
+        ArrayList<Entity> returnMetaData = new ArrayList<>();
+        ArrayList<ModuleMemberDeclarationNode> returnModuleMembers = new ArrayList<>();
```

Comment:
```suggestion
        ArrayList<Entity> entities = new ArrayList<>();
        ArrayList<ModuleMemberDeclarationNode> moduleMembers = new ArrayList<>();
```
What are we storing in moduleMembers.

Reply:
Store the entities as ModuleMemberDeclarationNodes.